hw/sch/power_board/CHANGELOG.md

```diff
@@ -7,3 +7,15 @@ Initial release
 * TH2/TL2 pulled high via 4.7k resistor.
 * TH1/TL1 pulled low via 4.7k resistor.
 * 9VA DC-DC supply removed, replaced with 9VA isolated supply ITR0305S09.
+
+## Rev 3.B
+* Power good resistors changed for 5V/12V DC-DC power supplies to make LEDs dimmer
+* 9VA DC-DC supply removed, replaced with 5VA and 9VA isolated supplies.
+* Switched all 3.3V circuitry over to analog ground as the preamp board no longer has digital ground.
+* R4 added to reduce PG_12V to 5V to 3.3V logic.
```

Comment:
R4 added to reduce PG_12V and PG_5V to 3.3V logic?

Reply:
Good catch, it was supposed to be "to reduce PG_12V from 5V to 3.3V logic.